packages/test-base/src/androidAndroidTest/kotlin/io/realm/kotlin/test/shared/dynamic/DynamicRealmObjectTests.kt

```diff
@@ -125,6 +127,77 @@ class DynamicRealmObjectTests {
         assertEquals("Sample", dynamicSample.type)
     }
 
+    @Test
+    fun get_realmAny() {
+        // It's not possible to integrate in get_allTypes() since RealmAny cannot be nullable
```

Comment:
Poor wording. I mean not nullable in the context of the test, as the test checks only default values. However, we have to test several supported types and going through the non-nullable path doesn't do anything since it's not supported at a schema level.

Reply:
Ok ... Would have been nice to update the wording then 🤪